cafebabel/templates/articles/macros.html

```diff
@@ -18,7 +18,7 @@ <h4>Words by:</h4>
       {% for author in article.authors %}
           <a href="{{ url_for('users.detail', id=author.pk) }}" class=author-link>
             {% if author.profile.has_image %}
```

Comment:
I would create a `author_image(author)` macro dedicated to that repeated case.

Reply:
ok: https://github.com/cafebabel/cafebabel.com/pull/305/commits/1e97685adf27432cb9f69d50180d84e62d4f925f